src/__tests__/App.test.js

```diff
@@ -1,3 +1,11 @@
 import { render, screen } from '@testing-library/react';
 import App from '../components/App';
 
+describe("app", () => {
+const { asFragment } = render(<App />)
+
+it("renders correctly", () => {
+   expect(asFragment()).toMatchSnapshot();
+});
+
```

Comment:
Was this test copy pasted in from somewhere else? Cant comment on the first line as its from the first commit and the PR doesnt cover it, but you should remove the `screen` import as it isnt being used in the file anywhere
```
  const { asFragment } = render(<App />);

  it("renders correctly", () => {
    expect(asFragment()).toMatchSnapshot();
  });
```